metric/distribution/distribution.go

```diff
@@ -5,13 +5,20 @@ package distribution
 
 import (
 	"errors"
+	"math"
 
 	"go.opentelemetry.io/collector/pdata/pmetric"
 )
 
+const (
+	epsilon = 0.001
+)
+
 var (
 	ErrUnsupportedWeight = errors.New("weight must be larger than 0")
-	ErrUnsupportedValue  = errors.New("value cannot be negative, NaN, or Inf")
+	ErrUnsupportedValue  = errors.New("value cannot be negative, NaN, Inf, or greater than 2^360")
+	MinValue             = -math.Pow(2, 360)
+	MaxValue             = math.Pow(2, 360)
```

Comment:
Could this be a `const` instead of `var`?

Reply:
No. `math.Pow` is a function, so it needs to be a var.